helm/kong-app/templates/pdb.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.podDisruptionBudget.enabled }}
+{{- if and .Values.podDisruptionBudget.enabled (or (and (not .Values.autoscaling.enabled) (gt (int .Values.replicaCount) 1)) (and .Values.autoscaling.enabled (gt (int .Values.autoscaling.minReplicas) 1)))}}
```

Comment:
Yes I agree, your suggestion looks better. I'll create a follow up PR..

Casting to `int` is required because of some helm bug when using default values and `gt`

```
Error: template: kong-app/templates/pdb.yaml:1:48: executing "kong-app/templates/pdb.yaml" at <gt (.Values.autoscaling.enabled | ternary .Values.autoscaling.minReplicas .Values.replicaCount) 1>: error calling gt: incompatible types for comparison
```